packages/api/package.json

```diff
@@ -14,7 +14,8 @@
     "node": ">=12"
   },
   "scripts": {
-    "develop": "concurrently \"yarn generate -w\" \"tsdx watch\"",
+    "develop": "tsdx watch --entry ./local/server.ts --target node --onSuccess \"node ./dist\"",
```

Comment:
You changed the `local` command mentioned in the PR description to `develop`, right?

Reply:
Yep! Forgot to update the PR description 😅  Sorry about that